hermes/src/api/rest/v2/sse.rs

```diff
@@ -105,10 +114,26 @@ pub async fn price_stream_sse_handler(
                         price_ids_clone,
                         params.encoding,
                         params.parsed,
+                        params.benchmarks_only,
+                        params.allow_unordered,
                     )
                     .await
                     {
-                        Ok(price_update) => Ok(Event::default().json_data(price_update).unwrap()),
+                        Ok(price_update) => {
+                            // Check if there is any data to send
+                            if let Some(update) = price_update {
+                                if update.binary.data.is_empty() {
```

Comment:
I think it might be better if we return a None if the list of price_ids becomes empty because of benchmarks.